### PR TITLE
feat(jut.su): took over ownership

### LINKED ISSUE #6569: Jut.su | Presence broken
### Steps to Reproduce

1. Add the presence from [the store](https://premid.app/store/presences/jut.su)
2. Go to the [website](https://jut.su/)
3. Presence does not show up on the Discord profile

### Expected Behavior

Presence should be set on the profile

### Actual Behavior

Presence does not show up on the profile

### Page URL

https://jut.su/

### Screenshots

image

Note: the warning "VM547:6773 crbug/1173575, non-JS module files deprecated." is reported even with the presence disabled.

### Operating System

Windows 10

### Browser and Version

Brave 1.41.100

### Additional Comments

The browser console does not report any errors related with the presence

### PR DESCRIPTION
**Pull request by DarkVillager**
## Description 

Resolves #6569
## Acknowledgements
- [x] I read the [Presence Guidelines](https://github.com/PreMiD/Presences/blob/main/.github/CONTRIBUTING.md)
- [x] I linted the code by running `yarn format`
- [x] The PR title follows the repo's [commit conventions](https://github.com/PreMiD/Presences/blob/main/.github/COMMIT_CONVENTION.md)

## Screenshots
<details>
<summary> Proof showing the creation/modification is working as expected </summary>

</details>

### CODE REVIEW

**Comment by RisingSunLight42** [comment]
Well, okay xD
I should use draft PR mord often when I plan doing something after few days if someone doesn't respond lol

File: websites/J/jut.su/dist/metadata.json [hunk 1]
```diff
@@ -11,7 +11,7 @@
 		"nl": "Jut.su is een Russische anime-website"
 	},
 	"url": "jut.su",
-	"version": "1.0.11",
+	"version": "1.1.0",
```
Comment (EncryptedDev) [inline websites/J/jut.su/dist/metadata.json:14]: Should be a major bump as you changed the Client ID 
```suggestion
	"version": "2.0.0",
```

**Review by EncryptedDev — CHANGES_REQUESTED** [review]

**Comment by EncryptedDev** [comment]
As the issue was opened fairly recently and the GitHub user still exists, there is still a chance to recieve a response from the OP. This should be kept as a draft for at least a week to make sure of this.

File: websites/J/jut.su/dist/metadata.json [hunk 2]
```diff
@@ -1,8 +1,8 @@
 {
 	"$schema": "https://schemas.premid.app/metadata/1.7",
 	"author": {
-		"name": "Yokkkoso && Baka",
-		"id": "190762645316567040"
+		"name": "Dark_Ville",
+		"id": "638080361179512853"
 	},
 	"service": "jut.su",
 	"description": {
```
Comment (Syjalo) [inline websites/J/jut.su/dist/metadata.json:8]:
```json
"ru": "Jut.su — это русский сайт аниме"
```

File: websites/J/jut.su/presence.ts [hunk 3]
```diff
@@ -1,103 +1,50 @@
...
+	else if (mangaTitle) {
+		presenceData.details = mangaTitle[0];
+		presenceData.state = mangaTitle[1];
+	} else if (video) {
+		delete presenceData.startTimestamp;
+
+		if (!video.paused && !isNaN(Number(video.duration))) {
+			presenceData.endTimestamp = presence.getTimestampsfromMedia(video)[1];
+			presenceData.smallImageKey = "play";
+			presenceData.smallImageText = "Игра";
```
Comment (Syjalo) [inline websites/J/jut.su/presence.ts:37]:
```suggestion
			presenceData.smallImageText = "Воспроизводится";
```

File: websites/J/jut.su/presence.ts [hunk 4]
```diff
@@ -1,103 +1,50 @@
...
+		} else {
+			delete presenceData.endTimestamp;
+			presenceData.smallImageKey = "pause";
+			presenceData.smallImageText = "Приостановлено";
 		}
-	}
+		const titles = name.textContent.replace(/[0-9]* сезон [0-9]* серия/g, "");
+		presenceData.details = titles;
+		presenceData.state = name.textContent.replace(titles, "");
+	} else if (title) presenceData.details = `Просмотр ${title.textContent}`;
```
Comment (Syjalo) [inline websites/J/jut.su/presence.ts:46]:
```suggestion
	} else if (title) presenceData.details = `Смотрит «${title.textContent}»`;
```

File: websites/J/jut.su/presence.ts [hunk 5]
```diff
@@ -1,103 +1,50 @@
...
+		mangaTitle = document
+			.querySelector("#the_manga_title")
+			?.textContent.split(":"),
+		video = document.querySelector<HTMLVideoElement>(
+			'[id="my-player_html5_api"]'
+		),
+		name = document.querySelector("#dle-content > div > h1 > span");
+	if (search?.value) {
+		presenceData.details = "идет поиск";
+		presenceData.state = search.value;
```
Comment (Syjalo) [inline websites/J/jut.su/presence.ts:25]:
```suggestion
		presenceData.details = "Ищет по запросу";
		presenceData.state = `«${search.value}»`;
```

**Review by Syjalo — CHANGES_REQUESTED** [review]

**Review by Syjalo — APPROVED** [review]

**Review by EncryptedDev — APPROVED** [review]

**Review by Bas950 — APPROVED** [review]

**Pull request merged**